io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/reader/internal/wfs/WfsBackedGmlInstanceCollection.java

```diff
@@ -497,6 +491,9 @@ private void createNextIterator() {
 					sourceSchema, restrictToFeatures, ignoreRoot, strict, ignoreNamespaces,
 					crsProvider, ioProvider);
 			iterator = currentCollection.iterator();
+			if (primordialQueryParams.containsKey("RESOLVEDEPTH")) {
```

Comment:
@stempler Could you please help me understand how to initialize the DuplicateIDsFilterIterator that it's not initialize with every new request for pagination? Thank you